TombEngine/Game/animation.cpp

```diff
@@ -501,3 +501,20 @@ Vector3i GetJointPosition(ItemInfo* item, int jointIndex, const Vector3i& offset
 	g_Renderer.GetItemAbsBonePosition(itemNumber, pos, jointIndex);
 	return Vector3i(pos);
 }
+
+SparkOffsets GetSparkOffset(Vector3i pos)
```

Comment:
This function does not belong to `animation.cpp`. Find better place, probably `effects.cpp`.

Reply:
I replaced it into effects.cpp